src/components/Api.js

```diff
@@ -0,0 +1,88 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.header = options.headers;
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+    return Promise.reject(`Error ${res.status}`);
+  }
+
+  getInitialCards() {
+    return fetch(this.baseUrl + "/cards", {
+      headers: this.header,
+    })
+      .then(this._checkResponse)
+      .then((result) => {
```

Comment:
Needs correction: please remove this `then` block as it does nothing with the response. Also fix that for other api methods.